mountpoint-s3/src/main.rs

```diff
@@ -287,6 +290,7 @@ fn mount(args: CliArgs) -> anyhow::Result<FuseSession> {
         part_size: args.part_size.map(|t| t as usize),
         endpoint,
         user_agent_prefix: Some(format!("mountpoint-s3/{}", build_info::FULL_VERSION)),
+        request_payer: args.request_payer
```

Comment:
Also add trailing comma here.

```suggestion
        request_payer: args.request_payer,
```